HHS/Head-Start-TTADP
# [TTAHUB-2647] Add goal name filter to training report

## Description of change
Add a goal name filter to the training report scopes (backend only)

## How to test
Evaluate the code. You can attach "logging: console.log" to the options passed to the EventReportPilot.findAlls added to the test to see the SQL generated if that helps evaluate the correctness. Ensure the test adequately covers the use case.

## Issue(s)

* https://jira.acf.gov/browse/TTAHUB-2647


## Checklists

### Every PR



### Before merge to main


### Production Deploy


### After merge/deploy
